server.go

```diff
@@ -2925,6 +2925,7 @@ func newServer(ctx context.Context, listenAddrs []string, db database.DB, chainP
 			AcceptSequenceLocks: s.chain.IsFixSeqLocksAgendaActive,
 		},
 		ChainParams: chainParams,
+		Net:         chainParams.Net,
```

Comment:
And then this would be replaced with `MaxVoteAge` up in the policy definition above to either the coinbase maturity or the longer `maximumVoteAgeDelta` (moved from `mempool.go` to `server.go` of course).

e.g.:
```Go
MaxVoteAge: func() uint32 {
	select chainParams.Net {
	case wire.MainNet, wire.SimNet, wire.RegNet:
		return chainParams.CoinbaseMaturity

	case wire.TestNet3:
		return maximumVoteAgeDelta
	}

	return chainParams.CoinbaseMaturity
},
```